docs/tutorial/tutorial.md

```diff
@@ -291,6 +291,12 @@ handleClick(i) {
 
 Now X and O take turns. Next, change the "status" text in Board's `render` so that it also displays who is next.
 
+```javascript
+render() {
+    const status = 'Next player:' + (this.state.xIsNext ? "X" : "O");
```

Comment:
In particular, this line of code already exists elsewhere in the tutorial, so we should just use the same one here. It doesn't use template strings but it does do a consistent single-quoting. I'll just fix up this PR directly rather than back-and-forthing around these nitpicks though.